client/src/store/modules/eventTypes/mutations.js

```diff
@@ -0,0 +1,20 @@
+import * as mutations from './types/mutations';
+
+export default {
+    [mutations.SET_EVENT_TYPES]: (state, eventTypes) => {
+        state.eventTypes = eventTypes;
+    },
+    [mutations.DISABLE_EVENT_TYPE_BY_ID]: (state, data) => {
+        const eventType = state.eventTypes.find(eventType => {
+            return eventType.id === data.id;
+        });
+        if (eventType) {
+            eventType.disabled = data.disabled;
+        }
```

Comment:
Actually, you mutate an array, `b = { a: 1 }; arr = [b]; arr[0].a = 2; b.a === 2` 

To change an array item use slice, here is an example
https://github.com/BinaryStudioAcademy/bsa-2020-schedulia/blob/develop/client/src/store/modules/status/mutations.js#L20

- index = findIndex
- if (index !== -1)
- state.eventTypes =  [...slice(0, index), {...eventTypes[index], disabled: data.disabled}, ...slice(index + 1)  ]